src/lib/translations/l10n-terra-components.ts

```diff
@@ -143,7 +143,8 @@ const de: {} = {
         configColumns: 'Spalten konfigurieren'
     },
     validators: {
-        required: 'Dies ist ein Pflichtfeld.'
+        required: 'Dies ist ein Pflichtfeld.',
+        pattern: 'Eingabe entspricht nicht dem Schema.'
```

Comment:
oder: "Eingabe muss mit dem gefordertem Format übereinstimmen."

Reply:
Am besten mal mit Johanna absprechen.

Reply:
Aber ggf. müsste in dem Text auch das geforderte Pattern angegeben werden. Sonst weiß der Nutzer immer noch nicht, wie er den Fehler vermeiden kann.